Tasks/CondaEnvironmentV0/task.json

```diff
@@ -29,17 +29,17 @@
             "name": "environmentName",
             "type": "string",
             "label": "Environment name",
-            "required": true,
+            "required": false,
             "defaultValue": "",
-            "helpMarkDown": "Name of the Conda environment to create and activate."
+            "helpMarkDown": "Name of the Conda environment to create and activate. If left empty, subsequent build steps will run in the global `base` environment."
```

Comment:
There's a new terminology decision to call these "build tasks" instead of "build steps."